src/component/CheckoutProduct.js

```diff
@@ -0,0 +1,108 @@
+import "./CheckoutProduct.css";
+import React, { useState, useEffect } from "react";
+
+import { useStateValue } from "../context/StateProvider";
+import Swal from "sweetalert2";
+
+function CheckoutProduct({
+  id,
+  title,
+  price,
+  category,
+  description,
+  image,
+  qty,
```

Comment:
use more descriptive varibale names that are easy to read and understand. Here 'quantity' wouldve been more readable. Small suggestion.